src/Bridge/Doctrine/MongoDbOdm/Extension/OrderExtension.php

```diff
@@ -58,7 +58,7 @@ public function applyToCollection(Builder $aggregationBuilder, string $resourceC
             if (empty($defaultOrder)) {
                 $defaultOrder = $this->resourceMetadataFactory->create($resourceClass)->getAttribute('order');
             }
-            if (\is_array($defaultOrder)) {
+            if (null !== $defaultOrder) {
```

Comment:
Isn't it better to keep `is_array`?

Reply:
I don't know, because:
- I consistently reverted all the changes of 3492922c2733de934cf4798893d907755640869d
- With `null !==` (like it was originally) an incorrect config (e.g. a string value) gives you a Warning "Invalid argument supplied for foreach()" (then you fix your config :wink:), with `is_array()` it is silently ignored (until you notice your config isn't applied?)
- See also #3472

But if you think this part of the revert is too risky, I can drop it

Reply:
Not sure which one is the better :sweat_smile: Keep it like this, it's alright.